srcs/cmd_functions.c

```diff
@@ -0,0 +1,43 @@
+#include "minishell.h"
```

Comment:
```suggestion
#include "minishell.h"
#include "libft.h"
```

Voir mon autre commentaire sur `minishell.h`.